js/react.md

```diff
@@ -238,26 +220,27 @@ const signUpConfig = {
       type: 'string'
     },
     {
-      label: 'Custom Attribute',
-      key: 'custom_attr',
-      required: false,
+      label: 'Email',
+      key: 'email',
+      required: true,
       displayOrder: 4,
-      type: 'string',
-      custom: true
+      type: 'string'
     }
   ]
 };
 const usernameAttributes = 'My user name';
 
-export default withAuthenticator(App, { 
-  signUpConfig, 
+export default withAuthenticator(App, {
+  signUpConfig,
   usernameAttributes
 });
 ```
 
 ## Add Analytics and Storage
 
-Next, we'll add some features, like tracking user behavior analytics and uploading/downloading images in the cloud. Start by running `amplify add analytics` in your project. You can enable analytics for authenticated users only, or for users that aren't authenticated. You would be prompted to ask whether you want to allow guests and unauthenticated users to send analytics events, so you can choose `Yes`. You can also try a new project without authentication configured to test this feature.
+Next, we'll add some features, like tracking user behavior analytics and uploading/downloading images in the cloud.
+
+Before starting, to enable analytics in amplify, we need to associate user to an identity pool. Run `amplify add auth` and configure authentication with using manual configuration. Then, Start by running `amplify add analytics` in your project. You can enable analytics for authenticated users only, or for users that aren't authenticated. You would be prompted to ask whether you want to allow guests and unauthenticated users to send analytics events, so you can choose `Yes`. You can also try a new project without authentication configured to test this feature.
```

Comment:
Mention of a gotcha around auth and identity pool so everyone can compile and run this without fails.

Reply:
I like this, but what about making it bullet points steps?